Buildings/Experimental/DHC/Networks/Connections/Examples/Connection2PipeExample.mo

```diff
@@ -0,0 +1,184 @@
+    redeclare package Medium = MediumW,
+    mDis_flow_nominal=mDis_flow_nominal,
+    mCon_flow_nominal=mCon_flow_nominal,
+    dhDisRet=dhRet,
+    lDis=lDis,
+    dhDis=dhDis)
+    annotation (Placement(transformation(extent={{20,40},{40,60}})));
+  Buildings.Fluid.Sources.Boundary_pT bouDisAutoSize(
+    redeclare final package Medium = MediumW,
+    p=300000,
+    nPorts=1) annotation (Placement(transformation(extent={{-50,10},{-30,30}})));
+  Modelica.Blocks.Sources.Constant mDis_flow_nominal_exp(k=mDis_flow_nominal)
+    annotation (Placement(transformation(extent={{-80,30},{-60,50}})));
+  Buildings.Experimental.DHC.EnergyTransferStations.BaseClasses.Pump_m_flow pumDisAutoSize(
+    dp_nominal=100000,
+    redeclare final package Medium = MediumW,
+    m_flow_nominal=mDis_flow_nominal) "Distribution network pump" annotation (
+      Placement(transformation(
+        extent={{-10,10},{10,-10}},
+        rotation=180,
+        origin={-10,20})));
+
+  Buildings.Experimental.DHC.EnergyTransferStations.BaseClasses.Pump_m_flow pumConAutoSize(
+    dp_nominal=5000,
+    redeclare final package Medium = MediumW,
+    m_flow_nominal=mDis_flow_nominal) "Agent connection pump" annotation (
+      Placement(transformation(
+        extent={{10,10},{-10,-10}},
+        rotation=180,
+        origin={30,82})));
+  Buildings.Experimental.DHC.Networks.Connections.Connection2PipePlugFlow connection2PipeplugFlow(
+    redeclare package Medium = MediumW,
+    mDis_flow_nominal=mDis_flow_nominal,
+    mCon_flow_nominal=mCon_flow_nominal,
+    dIns=0.001,
+    kIns=1,
+    lDis=lDis)
+    annotation (Placement(transformation(extent={{20,-60},{40,-40}})));
+  Buildings.Fluid.Sources.Boundary_pT bouDisPlugFlow(
+    redeclare final package Medium = MediumW,
+    p=300000,
+    nPorts=1)
+    annotation (Placement(transformation(extent={{-60,-90},{-40,-70}})));
+  Buildings.Experimental.DHC.EnergyTransferStations.BaseClasses.Pump_m_flow pumDisPlugFlow(
+    dp_nominal=100000,
+    redeclare final package Medium = MediumW,
+    m_flow_nominal=mDis_flow_nominal) "Distribution network pump" annotation (
+      Placement(transformation(
+        extent={{-10,10},{10,-10}},
+        rotation=180,
+        origin={-12,-80})));
+  Buildings.Experimental.DHC.EnergyTransferStations.BaseClasses.Pump_m_flow pumConPlugFlow(
+    dp_nominal=5000,
+    redeclare final package Medium = MediumW,
+    m_flow_nominal=mDis_flow_nominal) "Agent connection pump" annotation (
+      Placement(transformation(
+        extent={{10,10},{-10,-10}},
+        rotation=180,
+        origin={30,-20})));
+  Fluid.FixedResistances.BuriedPipes.PipeGroundCoupling           pipeGroundCoupling(
+    lPip=lDis,
+    rPip=0.04,
+    thiGroLay=1.1,
+    nSta=5,
+    nSeg=2)
+    annotation (Placement(transformation(extent={{-11,-10},{11,10}},
+        rotation=0,
+        origin={-9,-30})));
+
+  Modelica.Blocks.Sources.Step mCon_flow_nominal_exp(height=mCon_flow_nominal,
+      startTime=1800)
```

Comment:
Same as above, remove multiple empty spaces and make sure formatting is easy to read.

Reply:
I removed all the superfluous white space across the updated models